src/api/main.py

```
@@ -0,0 +1,142 @@
+from fastapi import FastAPI, Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
```

Ruff (0.11.9)

`fastapi.security.OAuth2PasswordBearer` imported but unused

Remove unused import

(F401)